README.md

```
@@ -114,9 +112,9 @@ npm run dev
 | `DB_NAME / DB_USER / DB_PASSWORD / DB_HOST / DB_PORT` | PostgreSQL connection |
 | `CORS_ALLOWED_ORIGINS` | Frontend origin |
 | `CSRF_TRUSTED_ORIGINS` | CSRF whitelist |
-Files: `.env.local` (dev) · `.env.testing` (tests) · `.env.production` (prod template)
+Files: `.env.local` (dev) · `.env.testing` (tests) · `.env.production.example` (production template) · `.env.vps` (server runtime, untracked)
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: no_leading_or_trailing; Missing leading pipe

(MD055, table-pipe-style)